sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -3,479 +3,624 @@
+}
 
-export { RetryOptions }
+export { RetryOptions };
 
 // @public
 export interface RuleDescription {
-    action?: string;
-    filter?: string | CorrelationFilter;
-    name: string;
+  action?: string;
+  filter?: string | CorrelationFilter;
+  name: string;
 }
 
 // @public
 export interface RuleDetails {
-    action?: SqlAction;
-    createdOn: string;
-    filter?: SqlFilter | CorrelationFilter;
-    ruleName: string;
-    subscriptionName: string;
-    topicName: string;
+  action?: SqlAction;
+  createdOn: string;
+  filter?: SqlFilter | CorrelationFilter;
+  ruleName: string;
+  subscriptionName: string;
+  topicName: string;
 }
 
 // @public
 export interface RuleOptions {
-    action?: SqlAction;
-    filter?: SqlFilter | CorrelationFilter;
+  action?: SqlAction;
+  filter?: SqlFilter | CorrelationFilter;
 }
 
 // @public
 export interface SendableMessageInfo {
-    body: any;
-    contentType?: string;
-    correlationId?: string | number | Buffer;
-    label?: string;
-    messageId?: string | number | Buffer;
-    partitionKey?: string;
-    replyTo?: string;
-    replyToSessionId?: string;
-    scheduledEnqueueTimeUtc?: Date;
-    sessionId?: string;
-    timeToLive?: number;
-    to?: string;
-    userProperties?: {
-        [key: string]: any;
-    };
-    viaPartitionKey?: string;
+  body: any;
+  contentType?: string;
+  correlationId?: string | number | Buffer;
+  label?: string;
+  messageId?: string | number | Buffer;
+  partitionKey?: string;
+  replyTo?: string;
+  replyToSessionId?: string;
+  scheduledEnqueueTimeUtc?: Date;
+  sessionId?: string;
+  timeToLive?: number;
+  to?: string;
+  userProperties?: {
+    [key: string]: any;
+  };
+  viaPartitionKey?: string;
 }
 
 // @public
 export class Sender {
-    cancelScheduledMessage(sequenceNumber: Long): Promise<void>;
-    cancelScheduledMessages(sequenceNumbers: Long[]): Promise<void>;
-    close(): Promise<void>;
-    get isClosed(): boolean;
-    scheduleMessage(scheduledEnqueueTimeUtc: Date, message: SendableMessageInfo): Promise<Long>;
-    scheduleMessages(scheduledEnqueueTimeUtc: Date, messages: SendableMessageInfo[]): Promise<Long[]>;
-    send(message: SendableMessageInfo): Promise<void>;
-    sendBatch(messages: SendableMessageInfo[]): Promise<void>;
-    }
+  cancelScheduledMessage(sequenceNumber: Long, options?: OperationOptions): Promise<void>;
+  cancelScheduledMessages(sequenceNumbers: Long[], options?: OperationOptions): Promise<void>;
+  close(): Promise<void>;
+  get isClosed(): boolean;
+  scheduleMessage(
+    scheduledEnqueueTimeUtc: Date,
+    message: SendableMessageInfo,
+    options?: OperationOptions
+  ): Promise<Long>;
+  scheduleMessages(
+    scheduledEnqueueTimeUtc: Date,
+    messages: SendableMessageInfo[],
+    options?: OperationOptions
+  ): Promise<Long[]>;
+  send(message: SendableMessageInfo, options?: OperationOptions): Promise<void>;
+  sendBatch(messages: SendableMessageInfo[], options?: OperationOptions): Promise<void>;
```

Comment:
This is incorrect and should be will be corrected to send a ServiceMessageBatch (equivalent of EventDataBatch in eventhubs)

Reply:
@HarshaNalluru has implemented this.